src/main/java/at/hannibal2/skyhanni/utils/ComputerTimeOffset.kt

```diff
@@ -56,6 +56,7 @@ object ComputerTimeOffset {
         val client = NTPUDPClient()
         val address = InetAddress.getByName(ntpServer)
         val timeInfo = client.getTime(address)
+        client.close()
```

Comment:
```suggestion
        val timeInfo = NTPUDPClient().use { client ->
            val address = InetAddress.getByName(ntpServer)
            client.getTime(address)
        }
```
(Requires updating `commons-net` to `3.11.1` in `build.gradle.kts`.)